core/extraction/towns_fund_round_one.py

```diff
@@ -983,6 +1016,9 @@ def transform_funding_questions(
 
     # merged_df.dropna(subset=merged_df.columns.difference(["Project ID"]), how="all", inplace=True)
 
+    unused_mask = merged_df.loc[merged_df["Programme ID"].str.contains(r"^HS")]
```

Comment:
I think a clearer(and probably faster) method here might be python str.startswith("HR") ?

Reply:
also, can you add a comment above to say what it's doing (ie, drop any Funding Questions for FHSF programmes)